client/task-list/style.scss

```diff
@@ -76,59 +76,66 @@
 	}
 
 	.woocommerce-task-dashboard__welcome-modal {
-		width: 400px;
+		max-width: 517px;
+		height: 460px;
 
-		.components-modal__header-heading-container {
-			justify-content: center;
-		}
+		.components-modal__header {
+			height: 0;
 
-		.components-modal__header-heading {
-			display: flex;
-			flex-direction: column;
-			align-items: center;
-			justify-content: center;
-			font-size: 20px;
+			.components-button {
+				color: $white;
+			}
 		}
 
-		.woocommerce-task-dashboard__welcome-modal-icon {
-			font-size: 56px;
-			margin-bottom: $gap;
-		}
+		.components-guide__container {
+			margin-top: 0;
 
-		.components-modal__header {
-			height: auto;
-			margin-bottom: $gap;
-			position: static;
-		}
+			.woocommerce-task-dashboard__welcome-modal__page-content {
+				padding: 0 24px;
 
-		.components-modal__content {
-			padding: $gap * 2;
-			text-align: center;
-		}
+				.woocommerce-task-dashboard__welcome-modal__page-content__header {
+					font-size: 24px;
+					line-height: 32px;
+					margin: 0 0 24px 0;
+				}
 
-		.woocommerce-task-dashboard__welcome-modal-message p {
-			font-size: 16px;
-			color: $studio-gray-60;
+				.woocommerce-task-dashboard__welcome-modal__page-content__body {
+					font-size: 16px;
+					line-height: 24px;
+					margin: 0 0 24px 0;
+				}
+			}
 		}
 
-		.components-modal__header .components-icon-button {
-			position: absolute;
-			left: auto;
-			right: 0;
-			top: 0;
-			padding: $gap;
+		.components-guide__footer {
+			padding: 0;
+			margin: 0 24px 24px 24px;
+			justify-content: flex-end;
+			width: initial;
+
+			.components-guide__finish-button,
+			.components-guide__forward-button,
+			.components-guide__back-button {
+				position: initial;
```

Comment:
There isn't a lot of control you can have over the rendering of the buttons in the Guide, so I had to mimic styles from other button types like `secondary` in the styles to match the designs.